notebooks/learning_graph_structured_data_h2.ipynb

```diff
@@ -0,0 +1,835 @@
+{
```

Comment:
<p>Ah, the definition comes here! (See my optional remark above).</p><p> </p><p>It would have been great if the geodesics plotted above were plotted with <code>geomstats</code>,  but we can also do this later.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
@Hzaatiti  If there's time at the end, let's do this!